packages/dendron-cli/src/commands/workspaceCLICommand.ts

```diff
@@ -84,7 +84,7 @@ export class WorkspaceCLICommand extends CLICommand<
         }
         case WorkspaceCommands.ADD_AND_COMMIT: {
           const ws = new WorkspaceService({ wsRoot });
-          await ws.commitAndAddAll();
+          await ws.commitAndAddAll({ engine: engine! });
```

Comment:
instead of force casting engine, can we add a check if engine exists and show error if thats the case?